flightcode/video: use gstreamer-1.0 pkg-config and allow PKG_CONFIG override

The original SoloLink build scripts expect GStreamer 0.10 and call pkg-config
directly. On Scarthgap, we build against GStreamer 1.0 and rely on Yocto's
native pkg-config wrapper.

--- a/flightcode/video/vid/Makefile
+++ b/flightcode/video/vid/Makefile
@@ -12,1 +12,2 @@
-PKGCONFIG = `pkg-config gstreamer-0.10 --cflags --libs`
+PKG_CONFIG ?= pkg-config
+PKGCONFIG = `$(PKG_CONFIG) gstreamer-1.0 --cflags --libs`

--- a/flightcode/video/hdmi/Makefile
+++ b/flightcode/video/hdmi/Makefile
@@ -12,1 +12,2 @@
-PKGCONFIG = `pkg-config gstreamer-0.10 --cflags --libs` 
+PKG_CONFIG ?= pkg-config
+PKGCONFIG = `$(PKG_CONFIG) gstreamer-1.0 --cflags --libs`

